cmd/dev.go

```diff
@@ -434,5 +444,73 @@ func (cmd *DevCmd) loadConfig(generatedConfig *generated.Config) *latest.Config
 		log.Fatalf("Couldn't save generated config: %v", err)
 	}
 
+	// Adjust config for interactive mode
+	if cmd.Interactive != "" {
+		if config.Images == nil {
```

Comment:
There should be also a check for len(*config.Images) == 0, since it is possible to have an empty map for images